archive/oldchangelog.html

```diff
@@ -208,7 +208,7 @@
 	  table. (Needed because of a bug which introduced garbage).
 	<LI>
 	  We didn't think we could read in a kerning subtable (class or pair) with
-	  device tables, so we converted them to pst data. We can read them in in optimized
+	  device tables, so we converted them to pst data. We can read them in optimized
```

Comment:
Can be read as akin to "We can read them in(to memory) in optimized form ..." These constructs may be seen less often now, but were not non-standard.

And if I'm reading the filename correctly, you are trying to edit an old changelog? *Really*!?? Methinks 'automatic' is overstated here?